robot/rover/Arm/Arm.ino

```diff
@@ -1,894 +1,107 @@
-//    if (isHoming) { // not done homing the motors
-//        homeArmMotors(); // Homing functionality ignores most message types. This function behaves differently each loop
-//    }
+  // tx2 sends to 25,     output 26
+  // TX teensy, RX teensy, enable pin, transmit pin
+  commandCenter->startSerial(1, 0, 25, 26);
 
-    if(Serial.available() > 0) {
-        commandCenter->readCommand();
-    }
-
-   commandCenter->sendMessage();
-} // end of loop
+  digitalWrite(LED_BUILTIN, LOW);
+}
 
-/*! Each motor with an encoder needs to attach the encoder and 2 interrupts.
-  This function also sets pid parameters and joint angle tolerances.
+/**
+ * Implements all the Arm commands.
  */
-void initEncoders(void) {
-
-    motor1.attachEncoder(M1_ENCODER_A, M1_ENCODER_B, M1_ENCODER_PORT, M1_ENCODER_SHIFT, M1_ENCODER_RESOLUTION);
-    attachInterrupt(motor1.encoderPinA, m1_encoder_interrupt, CHANGE);
-    attachInterrupt(motor1.encoderPinB, m1_encoder_interrupt, CHANGE);
-
-    motor2.attachEncoder(M2_ENCODER_A, M2_ENCODER_B, M2_ENCODER_PORT, M2_ENCODER_SHIFT, M2_ENCODER_RESOLUTION);
-    attachInterrupt(motor2.encoderPinA, m2_encoder_interrupt, CHANGE);
-    attachInterrupt(motor2.encoderPinB, m2_encoder_interrupt, CHANGE);
-
-    motor3.attachEncoder(M3_ENCODER_A, M3_ENCODER_B, M3_ENCODER_PORT, M3_ENCODER_SHIFT, M3_ENCODER_RESOLUTION);
-    attachInterrupt(motor3.encoderPinA, m3_encoder_interrupt, CHANGE);
-    attachInterrupt(motor3.encoderPinB, m3_encoder_interrupt, CHANGE);
 
-    motor4.attachEncoder(M4_ENCODER_A, M4_ENCODER_B, M4_ENCODER_PORT, M4_ENCODER_SHIFT, M4_ENCODER_RESOLUTION);
-    attachInterrupt(motor4.encoderPinA, m4_encoder_interrupt, CHANGE);
-    attachInterrupt(motor4.encoderPinB, m4_encoder_interrupt, CHANGE);
-
-    // set activate PIDs
-    motor1.isOpenLoop = false; //motor1.encoderModifier=-1;
-    motor2.isOpenLoop = false;
-    motor3.isOpenLoop = false;
-    motor4.isOpenLoop = false; //motor4.encoderModifier=-1;
-
-    // set pid gains
-    motor1.pidController.setGainConstants(10.0, 0.0, 0.0);
-    motor2.pidController.setGainConstants(10.0, 0.0, 0.0);
-    motor3.pidController.setGainConstants(10.0, 0.0, 0.0);
-    motor4.pidController.setGainConstants(10.0, 0.0, 0.0);
-
-    // set motor shaft angle tolerances
-    motor1.pidController.setJointAngleTolerance(0.1);//2.0 * motor1.gearRatioReciprocal); // randomly chosen for dc
-    motor2.pidController.setJointAngleTolerance(0.1);//2.0 * motor2.gearRatioReciprocal);
-    motor3.pidController.setJointAngleTolerance(0.1);//2.0 * motor3.gearRatioReciprocal);
-    motor4.pidController.setJointAngleTolerance(0.1);//2.0 * motor4.gearRatioReciprocal);
-    motor5.pidController.setJointAngleTolerance(0.1);//2.0 * motor5.gearRatioReciprocal); // randomly chosen for servo
-    motor6.pidController.setJointAngleTolerance(0.1);//2.0 * motor6.gearRatioReciprocal);
+void invalidCommand(const uint8_t cmdID, const uint8_t* rawArgs,
```

Comment:
This is nice, but it will trigger for every single command that is for the wheels since the arm teensy will jump to the the default statement in the switch case for the command ids used by the wheels.